tools/base/fpgaconf/fpgaconf.c

```diff
@@ -95,8 +96,8 @@ struct bitstream_info {
 	fpga_guid interface_id;
 };
 
-static fpga_result get_bitstream_ifc_id(const uint8_t *bitstream,
-					fpga_guid *guid)
+fpga_result get_bitstream_ifc_id(const uint8_t *bitstream,
```

Comment:
What about STATIC?

Reply:
It's not necessary.  Each of the other helper fn's in this file could have theoretically been declared static..  Since it's an app, I don't think it makes much difference.

Reply:
👍